portfolios/Kevin/Kevin.html

```diff
@@ -0,0 +1,191 @@
+            max-width: 200px;
+            border-radius: 10px;
+            margin-bottom: 10px;
+        }
+
+        .sub-section h3 {
+            margin-top: 10px;
+            font-size: 18px;
+        }
+
+        .sub-section p {
+            font-size: 16px;
+            margin-bottom: 10px;
+        }
+    </style>
+</head>
+<body>
+
+<div class="banner">
+    <a href="#inicio">
+        <img src="logo.png" alt="Logo de Kevin Hernández">
+    </a>
+    <div class="social-buttons">
+        <a href="https://www.instagram.com/kevinhdez242/" target="_blank"><img src="instagram.png" alt="Instagram"></a>
+        <a href="https://www.linkedin.com/in/kevinhernandezquiros/" target="_blank"><img src="linkedin.png" alt="LinkedIn"></a>
+        <a href="https://github.com/kevinhdez242" target="_blank"><img src="github.png" alt="GitHub"></a>
+    </div>
+</div>
+
+<div class="portada" id="inicio">
+    <img src="pfp.png" alt="Foto de Portada">
+    <h1>Kevin Hernàndez</h1>
+    <p>Bienvingut al meu portafoli!</p>
+</div>
+
+<div class="section">
+    <h2>Estudis</h2>
+    <div class="sub-sections">
+        <div class="sub-section">
+            <img src="paidos.png" alt="Escola Paidos">
+            <h3>Primaria i ESO</h3>
+            <p>(2010-2020)</p>
+        </div>
+
+        <div class="sub-section">
+            <img src="ies.png" alt="IES Castellet">
+            <h3>Cicle de Grau Mitjà SMIX</h3>
+            <p>(2021-2023)</p>
+        </div>
+
+        <div class="sub-section">
+            <img src="ies.png" alt="IES Castellet">
+            <h3>Cicle de Grau Superior DAM</h3>
+            <p>(2023-2025)</p>
+        </div>
+    </div>
+</div>
+
+<div class="section">
+    <h2>Pràctiques</h2>
+    <div class="sub-sections">
+        <div class="sub-section">
+            <img src="deodat.png" alt="Logo Lycée Déodat de Séverac">
+            <h3>Lycée Déodat de Séverac</h3>
+            <p>Vaig estar cursant pràctiques a Toulouse gràcies a la possibilitat que ens va oferir Erasmus+ per les pràctiques obligatòries del Cicle Mitjà.</p>
+        </div>
+
+        <div class="sub-section">
+            <img src="servitec.png" alt="Logo ServiTec Informàtica i Tecnologies">
+            <h3>ServiTec Informàtica i Tecnologies</h3>
+            <p>Quan vaig tornar de Toulouse, que va ser 1 mes, vaig continuar els pròxims 4 mesos a una botiga de tecnologia anomenada ServiTec a Manresa.</p>
+        </div>
+    </div>
+</div>
+
+<div class="section">
+    <h2>Projectes Personals</h2>
+    <div class="sub-sections">
+        <div class="sub-section">
+            <img src="allistix.png" alt="Proyecto 1">
+        </div>
+
+        <div class="sub-section">
+            <img src="primecost.png" alt="Proyecto 2">
+        </div>
+
+        <div class="sub-section">
+            <img src="unibuy.png" alt="Proyecto 3">
+        </div>
+
+        <div class="sub-section">
+            <img src="logo.png" alt="Proyecto 3">
+        </div>
```

Comment:
You are missing some closing tags at the end of this document

Reply:
Ja està ben tancat el document